tasks/process-spectrum.js

```diff
@@ -385,7 +374,8 @@ async function processComponent(componentPath) {
                         }
                     }
                 });
-                // @ts-ignore
+
+                // @ts-expect-error - host is defined
```

Comment:
if `host` is defined, why do we need to check for `if (host)`?

Reply:
Maybe a better comment is that the host variable _does_ exist because the ts error has to do with host not existing.

Reply:
Turns out no comment needed at all!